doc/source/api/evaluate/index.rst

```diff
@@ -16,15 +16,15 @@ to facilitate the manipulation and execution of Twin Runtimes through TwinModel.
 Workflow Example
 ----------------
 
-.. code-block:: python
+.. code-block:: pycon
```

Comment:
typo: python (or maybe not since it is used in other files?)

Reply:
@lboucin It is `pycon`. Actually with the new check of the documentation, pre-commit was failling.
[pycon](https://pvbookmarks.readthedocs.io/en/master/documentation/doc_generators/sphinx/rest_sphinx/code_block.html#pycon-python-console-highlighter) is for interactive Python highlighter.

Reply:
Ok thanks!